Create ui-c8y-1023-14-105-sso-users-confirm-with-code-when-suspend-or-delete-tenant.md (#4372)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-14-105-sso-users-confirm-with-code-when-suspend-or-delete-tenant.md

---
date: ""

> title: SSO users confirm with code when suspend or delete tenant

Reviewer: ```suggestion
title: SSO tenant management improved when suspending or deleting a tenant
```

> title: SSO users confirm with code when suspend or delete tenant

Reviewer: ```suggestion
title: SSO tenant management improved when suspending or deleting a tenant
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66119
version: 1023.14.105
---

> In a tenant that has SSO logon, if they try to suspend or delete a tenant, they are prompted for a password. SSO users authenticate withe external service so they are not able to provide Cumulocity account password. Now, instead of password, SSO users have to type randomly generated code to confirm suspend/delete action.

Reviewer: ```suggestion
When suspending or deleting a tenant in an SSO-enabled environment, the behavior has been changed from requiring a password to providing a randomly generated code that is entered in order to complete the confirmation step. This change ensures that SSO users can perform these critical tenant management operations without friction, while maintaining the same level of security through code-based confirmation as password-based confirmation provides for non-SSO users.
```
